<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlendTestDlgClass</class>
 <widget class="QDialog" name="BlendTestDlgClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BlendTestDlg</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,0,1" columnstretch="0,1">
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Make Simple Image</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="4" column="1" colspan="2">
       <widget class="QComboBox" name="cmbPattern">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <item>
         <property name="text">
          <string>입력된 값 사용</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Black</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>White</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>가로줄</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>세로줄</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Palette Index :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="chkColorImage">
        <property name="text">
         <string>Color Image</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Bit Per Pixel :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinImageWidth">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="minimum">
         <number>4</number>
        </property>
        <property name="maximum">
         <number>2147483647</number>
        </property>
        <property name="value">
         <number>800</number>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="3">
       <widget class="QCheckBox" name="chkIgnorePalette">
        <property name="text">
         <string>Pixel Index 값 그대로 출력
(RGB변환 안함, 메모리 절약모드)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>이미지 크기:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Pattern :</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="3">
       <widget class="QGroupBox" name="grpMarks">
        <property name="title">
         <string>Mark four corners</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="spinFontScale">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="minimum">
            <double>0.100000000000000</double>
           </property>
           <property name="maximum">
            <double>1000.000000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="spinFontThickness">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>scale :</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>thickness :</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="9" column="1" colspan="2">
       <widget class="QPushButton" name="btnSaveImageAs">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinImageHeight">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="minimum">
         <number>4</number>
        </property>
        <property name="maximum">
         <number>2147483647</number>
        </property>
        <property name="value">
         <number>600</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="cmbBPP">
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="spinPaletteIndex">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QPushButton" name="btnMakeImage">
        <property name="text">
         <string>Make</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="txtImageValue">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Test</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="2" column="0">
       <widget class="QPushButton" name="btnTest3">
        <property name="text">
         <string>Test3</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="btnTest1">
        <property name="text">
         <string>Test1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btnTest2">
        <property name="text">
         <string>Test2</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="btnTest4">
        <property name="text">
         <string>Test4</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="3">
    <widget class="gtl::qt::xMatView" name="view" native="true"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>gtl::qt::xMatView</class>
   <extends>QWidget</extends>
   <header>gtl/qt/MatView/MatView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>spinImageWidth</tabstop>
  <tabstop>cmbBPP</tabstop>
  <tabstop>cmbPattern</tabstop>
  <tabstop>spinPaletteIndex</tabstop>
  <tabstop>grpMarks</tabstop>
  <tabstop>chkIgnorePalette</tabstop>
  <tabstop>btnMakeImage</tabstop>
  <tabstop>btnSaveImageAs</tabstop>
  <tabstop>btnTest1</tabstop>
  <tabstop>btnTest3</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
